conformance/tests/ot_schema.py

```diff
@@ -26,5 +27,22 @@ def test_getscheama(man):
         errors.append("Incorrect labels returned from sampling: %s != %s " %
             (eLabels, eExpectedLabels)
         )
+    return errors
+
+
+def test_post_json_schema(man):
+    errors = []
+    G = man.setGraph("swapi")
+    # Probably don't want to add a 2MB schema file to the repo so get it via requests instead
+    res = requests.get("https://raw.githubusercontent.com/bmeg/iceberg/f1724941fe47df24846135fb515d1b89e791cee3/schemas/graph/graph-fhir.json")
```

Comment:
This probably isn't ideal in the long run. We can think about a paired down sample schema that can be used for unit tests in the future.